docker/crashpad_symbolize/Dockerfile

```diff
@@ -0,0 +1,63 @@
+FROM cobalt-base
+
+RUN apt update -qqy \
+    && apt install -qqy --no-install-recommends \
+        jq build-essential zlib1g-dev libzstd-dev \
+    && /opt/clean-after-apt.sh
+
+RUN cd /tmp \
+  && git clone 'https://chromium.googlesource.com/chromium/tools/depot_tools.git'
+
+ENV PATH=${PATH}:/tmp/depot_tools
+
+RUN mkdir breakpad \
+  && cd breakpad \
+  && fetch breakpad \
+  && cd src \
+  && ./configure \
+  && make
+
+ARG GITHUB_TAG
+RUN curl -L \
+  -H "Accept: application/vnd.github+json" \
+  -H "X-GitHub-Api-Version: 2022-11-28" \
+  https://api.github.com/repos/youtube/cobalt/releases/tags/${GITHUB_TAG} \
+  > /tmp/release.json
+
+ARG ARCHITECTURE
+ARG SB_API_VERSION
+ARG CONFIG
+ENV CONTAINS_STRING unstripped_${ARCHITECTURE}_sbversion-${SB_API_VERSION}_${CONFIG}
+RUN jq -r --arg str "$CONTAINS_STRING" '.assets[] | select(.name | contains($str)) | .id' /tmp/release.json > /tmp/asset_ids.txt \
+  && rm /tmp/release.json
+
+RUN while read asset_id; do \
+      curl -LJO \
+        -H "Accept: application/vnd.github+json" \
+        -H "X-GitHub-Api-Version: 2022-11-28" \
+        https://api.github.com/repos/youtube/cobalt/releases/assets/${asset_id}; \
+  done < /tmp/asset_ids.txt
```

Comment:
AFAICT the previous step generates multiple asset id's, while the following step acts as if that list is a single-item. Not sure if I understand what this step is doing, do you mind clarifying it for me?

Reply:
It's only a single asset id and could probably be done with a bash variable instead, I just found this easier